src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1042,4 +1042,7 @@
   <data name="LocalDBNotSupported" xml:space="preserve">
     <value>LocalDB is not supported on this Platform.</value>
   </data>
+  <data name="PlatformNotSupported_DataSqlClient" xml:space="preserve">
+    <value>System.Data.SqlClient is not supported on this platform.</value>
+  </data>
```

Comment:
Do I understand right then that we don't expect any consumers of this package would actually hit this? If so I don't care that much about the message, what is here is fine.